README.md

```diff
@@ -116,7 +130,9 @@ body, that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$("#greeting").on('click', function hello(){
+  $("body")append("<p>Hello</p>");
+})
```

Comment:
nice! I recommend either explicitly naming your function and defining it elsewhere, or use an anonymous function as callback to your `.on()` method call here